dohq_artifactory/admin.py

```diff
@@ -93,9 +94,14 @@ def _create_and_update(self):
             headers={"Content-Type": "application/json"},
             auth=self._auth,
         )
-        r.raise_for_status()
-        rest_delay()
-        self.read()
+        try:
+            r.raise_for_status()
+        except requests.exceptions.HTTPError as e:
```

Comment:
This exception (`HTTPError`) is too broad. If we talk about really a artifactory's error we should verify that it's not a 500 or higher error.

For the first iteration, let's except  `>400` error like metioned in https://github.com/devopshq/artifactory/issues/145 and re-raise a error otherways